scipy/stats/tests/test_mstats_basic.py

```diff
@@ -332,6 +332,14 @@ def test_kendalltau(self):
                             25, 80, 80, 80, 80, 80, 80, 0, 10, 45, np.nan, 0])
         result = mstats.kendalltau(x, y)
         assert_almost_equal(np.asarray(result), [-0.1585188, 0.4128009])
+
+        # test for namedtuple attributes
+        attributes = ('correlation', 'pvalue')
+        check_named_results(result, attributes, ma=True)
+
+
+    @pytest.mark.slow
```

Comment:
You lost the `ppc64le` skip here, let's put that back.

Reply:
Oops, good catch.